packages/mobile/src/harmony-native/utils/styleProps.ts

```diff
@@ -0,0 +1,25 @@
+import type { SpacingOptions } from '@audius/harmony'
+import { css } from '@emotion/native'
+import { useTheme } from '@emotion/react'
+
+export type MarginProps = {
+  m?: SpacingOptions
+  mt?: SpacingOptions
+  mr?: SpacingOptions
+  mb?: SpacingOptions
+  ml?: SpacingOptions
+  mx?: SpacingOptions
+  my?: SpacingOptions
+}
+export const useMargin = <T extends MarginProps>(props: T) => {
+  const { spacing } = useTheme()
+  return css({
+    ...(props.m && { margin: spacing[props.m] }),
+    ...(props.mt && { marginTop: spacing[props.mt] }),
+    ...(props.mr && { marginRight: spacing[props.mr] }),
+    ...(props.mb && { marginBottom: spacing[props.mb] }),
+    ...(props.ml && { marginLeft: spacing[props.ml] }),
+    ...(props.mx && { marginHorizontal: spacing[props.mx] }),
+    ...(props.my && { marginVertical: spacing[props.my] })
+  })
+}
```

Comment:
I wanted this to not be a hook but I dont think `emotion/native` does the same "inline theme" fn that the web one does
i.e. I tried to do it like this and it didnt work 😭 
```
return css(({spacing}) => ({...etc})
```